notebooks/official/ray_on_vertex/ray_cluster_management.ipynb

```diff
@@ -0,0 +1,638 @@
+{
```

Comment:
<p>how to use Ray on Vertex SDK &gt; how to use <strong>the</strong> Ray on Vertex <strong>AI</strong> SDK</p><p>Ray on Vertex overview &gt; Ray on Vertex <strong>AI</strong> overview</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
done